src/components/queue/QueueSearch.js

```diff
@@ -29,6 +29,11 @@ class QueueSearch extends Component {
           <View style={styles.container}>
             <View style={styles.topBar}>
               <Text style={styles.title}>Queue Search</Text>
+              <TouchableOpacity
+                style={styles.joinButton}
+                onPress={() => this.props.navigation.navigate('QueueScreen')}>
+                    <Text style={styles.joinButtonText}>Back to Queue Search</Text>
```

Comment:
Shouldn't this be "Back to Queue"?

Reply:
rename branch ("F52") and maybe make settings button flex row with location so its next to the .location. Like you can put the entire header container as display flex, then its child elements would have flexDirection: 'row';